test/civ2.test.js

```diff
@@ -0,0 +1,699 @@
+const chai = require('chai');
+const sinon = require('sinon');
+const Robot = require('../src/civ2');
+const civ2Commands = require('../src/lib/civ2-commands');
+const PivotalTracker = require('../src/lib/pivotal-tracker');
+const { setVariables, resetVariables } = require('./utils');
+
+chai.use(require('sinon-chai'));
+
+const { expect } = chai;
+
+describe('hubot integration', () => {
+  describe('Regisrations', () => {
+    let robot;
+
+    beforeEach(function() {
```

Comment:
Why all the `function` instead of arrows in this file?

Reply:
The weight of history, my dear :)